### DISCUSSION TITLE
Are there any cheat sheets for Altair or Vega-lite?

### DISCUSSION THREAD

**Discussion by joelostblom** [vega/altair#3728] [General]
Are there any cheat sheets or similar resources for Altair or Vega-lite [similar to this one for ggplot2](https://github.com/rstudio/cheatsheets/blob/master/data-visualization-2.1.pdf)? When I searched I could only find [this one](https://cheatography.com/gerog/cheat-sheets/python-altair/), but it is not very comprehensible. I might try to make one if I can find the time, but wanted to check so that I had not missed any existing ones first.

**Comment by jakevdp** [comment]
I don't know of any - if you do make one, please link to it here!
